tests/baselines/reference/bivariantInferences.symbols

```diff
@@ -24,8 +24,8 @@ declare const b: (string | number)[] | null[] | undefined[] | {}[];
 
 let x = a.equalsShallow(b);
 >x : Symbol(x, Decl(bivariantInferences.ts, 9, 3))
->a.equalsShallow : Symbol(Array.equalsShallow, Decl(bivariantInferences.ts, 2, 20))

```

Comment:
Yeah, it is. We dedupe less now because we use `===` to compare the symbol types (which is probably only true for this less instantiations of the same symbol), rather than `isIdenticalTo`.